src/items/tables/complete_species_table.json

```
@@ -1,7 +1,7 @@
 {
   "_id": "33udEssZ0Nk7jqtC",
   "name": "Complete Species table",
-  "description": "This table can be used to create random species tables for locations.\nOr as a base list to create Reincarnation lists from.",
+  "description": "This table can be used to create random species tables for locations.\nOr to create custom species tables for the Reincarnation spell or ritual.\nSubtype species results are accepted at the GM's discretion.",
```

```suggestion
  "description": "This table can be used to create random species tables for locations, or to create custom species tables for the Reincarnate spell or Reincarnation ritual.\nSubtype species results are accepted at the GM's discretion.",
```
Combine the first two sentences, and the spell and ritual have slightly different names